content/en/boilerplates/revision-tags-middle.md

```diff
@@ -12,10 +12,10 @@ The cluster operator can view this mapping in addition to tagged namespaces thro
 {{< text bash >}}
 $ istioctl tag list
 TAG         REVISION NAMESPACES
-prod-canary 1-10-0   ...
-prod-stable 1-9-5    ...
+prod-canary 1-10-0...
+prod-stable 1-9-5...
```

Comment:
Why are you removing the space before the `...`? Isn't the `...` supposed to be for the `NAMESPACES` column of the output?